### DISCUSSION TITLE
Link2Acad requirement

### DISCUSSION THREAD

**Discussion by a-nejjari** [wtertinek/Linq2Acad#37] [General]
Hi developers,

When I used this tool it gave me the next error:

"Could not load file or assembly 'Acdbmgd, Version=24.3.0.0, Culture=neutral, PublicKeyToken=null"

Does this tool need autocad software or something else? 

Thanks

**Comment by benkoshy** [comment]
Looks like you need to load the .net dlls. Check out the autocad .net start up guides.

**Reply by a-nejjari** [reply]
I don't have anything from autocad on my computer.

**Reply by benkoshy** [reply]
You need it. Otherwise it won't work.

**Reply by a-nejjari** [reply]
Ok thanks.
